io/eolearn/io/osm.py

```diff
@@ -0,0 +1,75 @@
+"""
+Module containing tasks used for reading vector data from OpenStreetMap
+"""
+
+import geopandas
+import overpass
+
+from sentinelhub.constants import CRS
+from shapely.geometry import shape, box, mapping, Polygon
+
+from eolearn.core import EOTask, FeatureType
+
+
+class OSMInput(EOTask):
+    """ Use OpenStreetMap (OSM) data from an Overpass API as input to a VECTOR_TIMELESS feature.
+    In case of timeouts or too many requests against the main Overpass endpoint, find additional
+    endpoints at see other options https://wiki.openstreetmap.org/wiki/Overpass_API#Public_Overpass_API_instances
+
+    :param feature_name: EOPatch feature into which data will be imported
+    :type feature_name: (FeatureType, str)
+    :param query: Overpass API Querystring: https://wiki.openstreetmap.org/wiki/Overpass_API/Overpass_QL
+    :type query: str
+    :param polygonize: Whether or not to treat ways as polygons, defaults to True
+    :type polygonize: bool
+    :param overpass_opts: Options to pass to the Overpass API constructor,
+    see: https://github.com/mvexel/overpass-api-python-wrapper#api-constructor
+    :type overpass_opts: dict
+    """
+
+    def __init__(self, feature_name, query, polygonize=False, overpass_opts=None):
```

Comment:
change `feature_name` to `feature` for consistency with other tasks (please update docstring as well)